0128-energy-subnetwork.md

```diff
@@ -10,99 +10,97 @@
+We propose that 60B ENERGY be reserved for providing startup capital for Srcful AB and investors, required to launch the ENERGY subnetwork and develop the technical and operational infrastructure required to build the ENERGY subnetwork. The pool will have a lockup of a minimum of 12 months after token emission date. 190B ENERGY is emitted mainly as community rewards but also for the continued operations of the network. Community rewards are divided into two parts, the baseline rewards for participating in the subnetwork and the service rewards for participating in energy services. The baseline rewards will receive 15% of emitted ENERGY and the service rewards will receive 45%. Services are rewarded to reflect the respective DC burn of the DER host, remaining tokens are burned. We use net emissions to replenish burned tokens into service rewards using a 30-day average of burned tokens with a cap. The cap will initially be set to 1% of current emissions but will likely be subject to change.
 
 We propose using one pool for buffering issued baseline rewards:
 
-**Network growth pool** acts as a resource for growing the network using direct incentives to DER hosts. This pool will receive tokens from the baseline DER host rewards inversely proportional to the total ENERGY network DC burn as long as this is less than $100 000 USD. The proportion allocated to the pool is calculated by:
+**Network growth pool** acts as a resource for growing the network using direct incentives to DER hosts. This pool will receive tokens from the baseline DER host rewards inversely proportional to the total ENERGY subnetwork DC burn as long as this is less than $100 000 USD. The proportion allocated to the pool is calculated by:
 
-Network&nbsp;Growth&nbsp;Pool&nbsp;Proportion =&nbsp;0.5&nbsp;&minus;&nbsp;min(0.5,&nbsp;0.5&nbsp;*&nbsp;Total&nbsp;ENERGY&nbsp;Network&nbsp;DC&nbsp;burn&nbsp;/&nbsp;1e10)
+Network&nbsp;Growth&nbsp;Pool&nbsp;Proportion =&nbsp;0.5&nbsp;&minus;&nbsp;min(0.5,&nbsp;0.5&nbsp;\*&nbsp;Total&nbsp;ENERGY&nbsp;Subnetwork&nbsp;DC&nbsp;burn&nbsp;/&nbsp;1e10)
```

Comment:
I think it would be cleaner to use LaTeX formatting here.
```suggestion
$\text{Network Growth Pool Proportion} = 0.5 - \min\left(0.5, 0.5 \times \frac{\text{Total Energy Subnetwork DC burn}}{10000000000}\right)$
```
$\text{Network Growth Pool Proportion} = 0.5 - \min\left(0.5, 0.5 \times \frac{\text{Total Energy Subnetwork DC burn}}{10000000000}\right)$